src/test/rgw/sfs/test_rgw_sfs_sqlite_buckets.cc

```diff
@@ -73,6 +73,11 @@ void compareBucketRGWInfo(const RGWBucketInfo & origin, const RGWBucketInfo & de
 
 void compareBuckets(const DBOPBucketInfo & origin, const DBOPBucketInfo & dest) {
   compareBucketRGWInfo(origin.binfo, dest.binfo);
+  ASSERT_EQ(origin.deleted, dest.deleted);
+}
+
+bool randomBool() {
+    return 0 + (rand() % (1 - 0 + 1)) == 1;
```

Comment:
this is the post std11 version way to throw a coin:
(more random than rand())

```
std::random_device generator; 
std::uniform_int_distribution<int> distribution(0,1);
int number = distribution(generator);
```

Reply:
On this [PR](https://github.com/aquarist-labs/ceph/pull/41), I introduced a specific test suite for `SFSBucket`.
I think would be desirable cover all `SFSBucket` methods we are going to implement/fix with a test.

Reply:
Yep, not a big thing as we don't really need a superb random for the test, though.
It was just to create the test bucket not always with the same value, but you're right, better to do it the C++11 way.
Changed.

I don't get what you mean in your second comment.
This is for testing the sqlite interface. Is there anything that you think that is missing for being tested?
I agree we need more tests for `sfs` types and methods, but we can do that in another PR.

Once your PR is merged I can add a specific test for deleting a bucket.

Reply:
My second comment only to make you aware of the test suite dedicated to `SFSBucket`.
Still to be merged btw, so totally agree to defer tests for remove bucket on a new PR. 
Generally speaking, we should always try to cover newly implemented methods with a test calling the new code (sqlite tests are good, but they don't call all the new code).

Reply:
yup, I agree, will add a remove bucket test when that PR is merged